doc/content/source/materials/NEMLStressInput.md

```diff
@@ -0,0 +1,18 @@
+<!-- MOOSE Documentation Stub: Remove this when content is added. -->
```

Comment:
Sorry... you're going to have to actually document these!

Reply:
I got rid of this class but I did up date the documentation.